src/smartAuthorizationHelper.ts

```diff
@@ -37,6 +42,10 @@ export function getFhirResource(resourceValue: string, defaultHostname: string):
 }
 
 const logger = getComponentLogger();
+AWSXRay.setContextMissingStrategy(() => {
+    logger.warn(`missing X-Ray context`);
+});
+logger.warn(`Segment: ${AWSXRay.getSegment()?.toString()}`);
```

Comment:
Is this the recommended log level for this type of log?

Reply:
ops, it was for debugging. I need to remove them.

Reply:
I have removed the logs